framework/decode/vulkan_replay_dump_resources_common.h

```diff
@@ -94,6 +94,13 @@ enum class DumpResourceType
     kDispatchTraceRaysInlineUniformBufferDescriptor,
 };
 
+enum CanImageBeDumped
```

Comment:
I think this is the name for a calculation, not a status, can this be ImageDumpResult?